packages/web-app-files/src/composables/parentShare/useIncomingParentShare.ts

```diff
@@ -0,0 +1,34 @@
+import { buildShare } from '../../helpers/resources'
+import { useStore } from 'web-pkg/src/composables'
+import { computed, ref, unref } from '@vue/composition-api'
+import { useTask } from 'vue-concurrency'
+import { clientService } from 'web-pkg/src/services'
+
+export function useIncomingParentShare() {
+  const store = useStore()
+  const incomingParentShare = ref(null)
+  const sharesTree = computed(() => store.state.Files.sharesTree)
```

Comment:
Please use the getter for the sharesTree: `store.getters['Files/sharesTree']`.
Reason: In my experience that's a good idea for building backwards compatibility. If something from the state gets moved or removed you can write some logic in the getter to keep it compatible for a while and deprecate it. With state that's not possible. That being said I'd even recommend to never access vuex state directly (outside of the store module itself).